Marlin/src/pins/lpc1769/pins_BTT_SKR_E3_TURBO.h

```diff
@@ -201,6 +201,20 @@
 #define EXP1_09_PIN                        P0_16
 #define EXP1_10_PIN                        P2_08
 
+#if ENABLED(DWIN_CREALITY_LCD)
+//You have to rewire Displaycable, TX = P0_15, RX = P0_16 with LCD_SERIAL_PORT 1
```

Comment:
I'd recommend this be turned into an `#error` to alert users of the required wiring changes.